qvm/sys-audio.sls

```diff
@@ -24,10 +24,26 @@ prefs:
   - pci_strictreset: false
   - pcidevs:   {{ salt['grains.get']('pci_audio_devs', [])|yaml }}
   - template:  {{ salt['pillar.get']('qvm:sys-audio:template', 'fedora-30-xfce') }}
+service:
+  - enable:
+    - audiovm-audio-agent
 {%- endload %}
 
 {{ load(defaults) }}
 
+# Setup Qubes RPC policy for sys-audio
+# If sys-audio is sys-gui it's not needed
+# as sys-gui is a admin-global-rwx
+sys-audio-rpc:
+  file.managed:
+    - name: /etc/qubes/policy.d/30-sys-audio.policy
+    - contents: |
+        admin.Events            *   sys-audio     @adminvm                allow
+        admin.vm.List           *   sys-audio     @adminvm                allow
+        admin.vm.List           *   sys-audio     @tag:audiovm-sys-audio  allow   target=dom0
+        admin.vm.property.Get   *   sys-audio     @tag:audiovm-sys-audio  allow   target=dom0
+        admin.vm.feature.Get    *   sys-audio     @tag:audiovm-sys-audio  allow   target=dom0
```

Comment:
Please list specific properties and features it really uses and limit to them here in argument (instead of `*`). I recommend simply checking journalctl in dom0 after calling qvm-start-daemon in sys-audio.